### PR TITLE
Feature: Portforwarding examples, providers.json rewording

### PR DESCRIPTION
**Pull request by onebeastchris**
Some changes, summarized:
**providers.json**
- yeeted some dead hosts: sunrisenode (website under construction for over a month now)
- updated some instructions, e.g. moved freemcservers from full auto to just supported since they no longer support Geyser-Standalone & instead require clone-remote-port active
 -> still in contact with revivenode/trying to contact other OVH-issue affected hosts for an update
 
**portforwarding page**
 - OVH/OCI 
 - Docker/Pterodactyl
 - Windows/Mac/Linux firewall issues are now on a separate portforwarding page
 
**General formatting changes**
Sometimes also made, such as breaking up mile long sentences into sub-sections so i dont have to scroll vertically in intelliJ; should have no effect on the website.
Added a disclaimer that Geyser-Standalone is not a plugin
 
**unable to connect page**
While still needing to get a major overhaul, i've temporarily cleaned it up + added some updated info.

### CODE REVIEW

**Review by rtm516 — CHANGES_REQUESTED** [review]

File: _docs/geyser/portforwarding.md [hunk 1]
```diff
@@ -0,0 +1,130 @@
+---
+title: Portforwarding
```
Comment (rtm516) [inline _docs/geyser/portforwarding.md:2]: Change filename to `port-forwarding.md` and any other references
```suggestion
title: Port forwarding
```

File: _docs/geyser/portforwarding.md [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+Other ISPs may require you to pay extra for a static IP address. <br>
+As an alternative to portforwarding, you can use <a href="/geyser/playit-gg/">playit.gg</a> to create a tunnel.
+</div>
+
+### Windows
+To open a port on Windows, you will need to open the port through the Windows Firewall. There are multiple ways to do this:
+
+**Powershell** <br>
+To open a port on UDP (in our example, port 19132), run the following command in an administrator Powershell: <br>
+`New-NetFirewallRule -DisplayName "Geyser" -Direction Inbound -Protocol UDP -LocalPort 19132 -Action Allow` <br>
```
Comment (rtm516) [inline _docs/geyser/portforwarding.md:30]: Use block code not inline and specify lang of `powershell`

File: _docs/geyser/portforwarding.md [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+<div class="alert alert-info" role="alert">
+	Some ISPs (Internet Service Providers) block certain ports, or don't allow you to open ports (e.g. by using CGNAT, which doesn't allow you to open a port with a dynamic IP).
+Other ISPs may require you to pay extra for a static IP address. <br>
+As an alternative to portforwarding, you can use <a href="/geyser/playit-gg/">playit.gg</a> to create a tunnel.
+</div>
+
+### Windows
+To open a port on Windows, you will need to open the port through the Windows Firewall. There are multiple ways to do this:
+
+**Powershell** <br>
```
Comment (rtm516) [inline _docs/geyser/portforwarding.md:28]: Remove all `<br>` and use 2x newlines if you want a newline (whole document)

File: _docs/geyser/portforwarding.md [hunk 4]
```diff
@@ -0,0 +1,130 @@
...
+To open a port on UDP (in our example, port 19132), run the following command in an administrator Powershell: <br>
+`New-NetFirewallRule -DisplayName "Geyser" -Direction Inbound -Protocol UDP -LocalPort 19132 -Action Allow` <br>
+After running this command, a rule named "Geyser" was created to allow UDP traffic on port 19132. <br>
+
+**Windows Defender Firewall with Advanced Security** <br>
+1. Search for "Windows Defender Firewall with Advanced Security" in the start menu, and open it. [Image](https://cdn.discordapp.com/attachments/613194762249437245/1139289055612370964/image.png)
```
Comment (rtm516) [inline _docs/geyser/portforwarding.md:34]: Can we move these images to part of the repo just incase the files are removed from discord? (whole document)

File: _docs/geyser/portforwarding.md [hunk 5]
```diff
@@ -0,0 +1,130 @@
...
+2. Click on "Inbound Rules" in the left sidebar. [Image](https://cdn.discordapp.com/attachments/613194762249437245/1139291934930772049/image.png)
+3. Click on "New Rule..." in the right sidebar. [Image](https://cdn.discordapp.com/attachments/613194762249437245/1139291934930772049/image.png)
+4. Select "Port" as the rule type and click "Next". [Image](https://cdn.discordapp.com/attachments/613194762249437245/1139292384283349092/image.png)
+5. Select "UDP" and "Specific local ports", and enter the port you want to open (in our example, 19132). Click "Next". [Example](https://cdn.discordapp.com/attachments/613194762249437245/1139292567410843658/image.png)
+6. Select "Allow the connection" and click "Next". [Image](https://cdn.discordapp.com/attachments/1029700125636960356/1139292899805249586/image.png)
+7. Select the profiles you want to apply the rule to (e.g. "Domain", "Private", "Public"), and click "Next". [Example](https://cdn.discordapp.com/attachments/1029700125636960356/1139292899536805949/image.png)
+8. Enter a name for the rule (e.g. "Geyser"), and click "Finish". [Image](https://cdn.discordapp.com/attachments/1029700125636960356/1139292899192881202/image.png)
```
Comment (rtm516) [inline _docs/geyser/portforwarding.md:41]:
```suggestion
1. Search for "Windows Defender Firewall with Advanced Security" in the start menu, and open it. ([Image](https://cdn.discordapp.com/attachments/613194762249437245/1139289055612370964/image.png))
2. Click on "Inbound Rules" in the left sidebar. ([Image](https://cdn.discordapp.com/attachments/613194762249437245/1139291934930772049/image.png))
3. Click on "New Rule..." in the right sidebar. ([Image](https://cdn.discordapp.com/attachments/613194762249437245/1139291934930772049/image.png))
4. Select "Port" as the rule type and click "Next". ([Image](https://cdn.discordapp.com/attachments/613194762249437245/1139292384283349092/image.png))
5. Select "UDP" and "Specific local ports", and enter the port you want to open (in our example, 19132). Click "Next". ([Example](https://cdn.discordapp.com/attachments/613194762249437245/1139292567410843658/image.png))
6. Select "Allow the connection" and click "Next". ([Image](https://cdn.discordapp.com/attachments/1029700125636960356/1139292899805249586/image.png))
7. Select the profiles you want to apply the rule to (e.g. "Domain", "Private", "Public"), and click "Next". ([Example](https://cdn.discordapp.com/attachments/1029700125636960356/1139292899536805949/image.png))
8. Enter a name for the rule (e.g. "Geyser"), and click "Finish". ([Image](https://cdn.discordapp.com/attachments/1029700125636960356/1139292899192881202/image.png))
```

File: _docs/geyser/portforwarding.md [hunk 6]
```diff
@@ -0,0 +1,130 @@
...
+
+### Linux
+Different Linux distributions, even different VPS providers ship and configure different firewalls. In the following examples, we will use `19132` as the port to open, but you should replace this with the port you are using for Geyser.
+
+- `ufw` is a simple firewall front-end for iptables that is commonly used on Ubuntu and Debian. To open a port on UDP, run the following command: <br>
+`sudo ufw allow 19132/udp` <br>
```
Comment (rtm516) [inline _docs/geyser/portforwarding.md:47]: Use block code not inline and specify lang of `bash`

File: _docs/geyser/portforwarding.md [hunk 7]
```diff
@@ -0,0 +1,130 @@
...
+Then, reload the firewall with `sudo ufw reload`, and see all open rules with `sudo ufw status`. <br>
+Further helpful guides: [DigitalOcean](https://www.digitalocean.com/community/tutorials/how-to-setup-a-firewall-with-ufw-on-an-ubuntu-and-debian-cloud-server), [Baeldung](https://www.baeldung.com/linux/uncomplicated-firewall)
+
+- `firewalld` Add a port on UDP by running <br>
+  `sudo firewall-cmd --zone=public --permanent --add-port=19132/udp` <br>
```
Comment (rtm516) [inline _docs/geyser/portforwarding.md:52]: Use block code not inline and specify lang of `bash`

File: _docs/geyser/portforwarding.md [hunk 8]
```diff
@@ -0,0 +1,130 @@
...
+  Then, reload the firewall with `sudo firewall-cmd --reload`, and see all open rules with `sudo firewall-cmd --list-all`. <br>
+  Further helpful guides: [DigitalOcean](https://www.digitalocean.com/community/tutorials/how-to-set-up-a-firewall-using-firewalld-on-centos-7)
+
+- `iptables` is a common firewall that is used on many Linux distributions. To open a port on UDP, run the following command: <br>
+`sudo iptables -A INPUT -p udp --dport 19132 -j ACCEPT` <br>
```
Comment (rtm516) [inline _docs/geyser/portforwarding.md:57]: Use block code not inline and specify lang of `bash`

**Review by rtm516 — CHANGES_REQUESTED** [review]

File: _docs/geyser/portforwarding.md (hunk 4 above)
Comment (onebeastchris) [reply _docs/geyser/portforwarding.md]: Done, created an /img/guides/port-forwarding folder for these

**Review by rtm516 — APPROVED** [review]

**Pull request merged**